frontend/src/types/transactions.ts

```diff
@@ -0,0 +1,12 @@
+export const enum TransactionType {
+    PAYMENT = "PAYMENT",
+    TOP_UP = "TOP_UP"
+}
+
+export interface Transaction {
```

Comment:
Maybe you missed it, but It's written that add ing'I' in the beginning is a bad practice.
But since you started using "i" everywhere continue adding it for this project to not spend time for refactoring